tests/CMakeLists.txt

```diff
@@ -123,12 +124,7 @@ if(AVIF_ENABLE_GTEST)
     add_avif_gtest(avifgridapitest)
     add_avif_gtest_with_data(avifilocextenttest)
     add_avif_gtest(avifimagetest)
-
-    add_executable(avifincrtest gtest/avifincrtest.cc)
-    target_link_libraries(avifincrtest aviftest_helpers avifincrtest_helpers)
-    add_test(NAME avifincrtest COMMAND avifincrtest ${CMAKE_CURRENT_SOURCE_DIR}/data/)
-    register_test_for_coverage(avifincrtest ${CMAKE_CURRENT_SOURCE_DIR}/data/)
-
+    add_avif_gtest_with_data(avifincrtest avifincrtest_helpers)
```

Comment:
Optional: This cleanup change is independent of the bug fix and can be moved to a separate pull request.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2317